### ISSUE TITLE
Where is the distillation loss?

### ISSUE THREAD

**Issue by yuchenlichuck** [mahtabbigverdi/Aurora-perception#13]
Hi, there habibi, I checked the paper and you claims there is an distillation loss, but I check the llava training code, i didn't find the it, could you tell me where is the loss function please
